gateway/transaction_test.go

```diff
@@ -0,0 +1,189 @@
+package gateway
+
+import (
+	"testing"
+	"context"
+	"math/big"
+)
+
+func TestGetTransaction(t *testing.T) {
```

Comment:
```suggestion
func TestTransaction(t *testing.T) {
```

Minor suggestion to remove `Get` to match function signature we're testing.

https://go.dev/doc/effective_go#Getters